backstop/tests/header.html

```diff
@@ -804,92 +917,100 @@ <h2>
+          <div class="iui-page-header-left">
             <!-- Application Button -->
             <div
-              class="iui-header-logo"
+              class="iui-header-brand"
               role="button"
               aria-label="App name"
-              aria-expanded="false"
               tabindex="0"
             >
               <svg-imodel-hollow
-                class="iui-header-button-icon"
+                class="iui-header-brand-icon"
                 aria-hidden="true"
               ></svg-imodel-hollow>
             </div>
 
-            <div class="iui-divider"></div>
+            <div class="iui-page-header-divider"></div>
 
-            <!-- Project Button -->
-            <nav aria-label="breadcrumbs">
-              <span class="iui-button-split-menu iui-header-split-button">
-                <div>
-                  <button
-                    class="iui-button iui-borderless iui-header-button"
-                    type="button"
-                    aria-label="Project picker"
-                    aria-expanded="false"
-                    id="test-button-1"
-                  >
-                    <div class="iui-button-icon iui-header-button-icon demo-picture"></div>
-                    <span class="iui-button-label">
-                      <div>Project Alpha</div>
-                      <div class="iui-description">0x0123456789</div>
-                    </span>
-                  </button>
-                </div>
-                <div>
-                  <button
-                    class="iui-button iui-borderless iui-header-button "
-                    id='test-button-6'
-                  >
-                    <svg-caret-down-small
-                      class="iui-button-icon"
-                      aria-hidden="true"
-                    ></svg-caret-down-small>
-                  </button>
-                </div>
-              </span>
-
-              <svg-chevron-right
-                class="iui-chevron"
-                aria-hidden="true"
-              ></svg-chevron-right>
-
-              <!-- IModel Button -->
-              <span class="iui-button-split-menu iui-header-split-button iui-active">
-                <div>
-                  <button
-                    class="iui-button iui-borderless iui-header-button"
-                    type="button"
-                    aria-label="Model picker"
-                    aria-expanded="false"
+            <nav
+              class="iui-header-breadcrumbs"
+              aria-label="Breadcrumbs"
+            >
+              <ol class="iui-breadcrumbs-list">
+                <li class="iui-breadcrumbs-item">
+                  <span class="iui-header-breadcrumb-button-container">
+                    <button
+                      class="iui-header-breadcrumb-button"
+                      type="button"
+                      aria-label="Project Alpha"
+                      id="test-button-5"
+                    >
+                      <img
```

Comment:
so now we support only `img` and `svg` ? div with bacgkround image was not working well?

Reply:
As I understand it the header in React is not using the `<div>` with background image.  That's part of the reason why https://github.com/iTwin/iTwinUI-react/issues/447 wasn't caught, because we weren't testing `<img>` on our HTML demo page.

Reply:
oh.. it is just an example..
We let user provide any of the following:
- `svg`
- `img`
- `div` with background image

So if it easy to support all these cases, lets have a test care with all of them.

Reply:
Added support for `<div>` with `background-image`.  Will add a test in my next push.

Reply:
This demo has been updated to show Version Gamma.  This tests against:
- `<img>`
- `<svg>`
- `<svg>` in active state.
- `<div>` with `background-image.

And all can be tested in disabled state as well.

![Screen Shot 2022-05-23 at 4 12 08 PM](https://user-images.githubusercontent.com/849817/169898399-9bb8f780-8e10-4e67-a0ca-cff9e6649345.png)